topydo/commands/EditCommand.py

```diff
@@ -73,12 +78,20 @@ def _todos_from_temp(self, p_temp_file):
 
         return todo_objs
 
-    def _open_in_editor(self, p_temp_file, p_editor):
+    def _open_in_editor(self, p_file):
```

Comment:
This change seems unrelated to the rest of the change. I'd suggest to put that in a different commit.